cmd/meroxa/root/connectors/remove.go

```diff
@@ -63,7 +63,7 @@ func (r *Remove) Execute(ctx context.Context) error {
 		return err
 	}
```

Comment:
You also don't need to retrieve the connector name above anymore https://github.com/meroxa/cli/pull/217/files#diff-51d616b524f4dc5671edd4e71797ce6a7cf9c1c23775b1d85c069ab03fd29efcR61

Reply:
this link isn't working for me

Reply:
there were a couple of remove functions that return the whole object... i didn't change that pattern. i don't know if anyone has come to rely on that pattern by now.